config/api.yml

```diff
@@ -770,6 +772,15 @@
       :post:
       - :name: assign
       - :name: unassign
+  :container_images:
```

Comment:
@ilackarms it looks like you just need to move this section to https://github.com/ManageIQ/manageiq-api/pull/52/files#diff-15502b585d5033aa8043cc26237a06baL686 to get this to 💚

Reply:
can you explain what you mean? shouldn't `  :container_images:` be a `collection`?

Reply:
currently the api endpoint is `/api/providers/<provider_id/container_images/<image_id>`

Reply:
nevermind, i see what you mean :+1: